statesman-engine/src/main/java/io/appform/statesman/engine/handlebars/HandleBarsHelperRegistry.java

```diff
@@ -328,6 +329,31 @@ private void registerAdd() {
         });
     }
 
+    private void registerSum() {
+        handlebars.registerHelper("sum", (Helper<Number>) (aNumber, options) -> {
+            val option = options.param(0);
+            Object[] params = options.params;
+            if (option instanceof Long) {
+                Long[] array = Arrays.asList(params).toArray(new Long[0]);
+                return aNumber.longValue() + Arrays.stream(array).reduce((long) 0, Long::sum);
+            }
+            if (option instanceof Integer) {
+                Integer[] array = Arrays.asList(params).toArray(new Integer[0]);
```

Comment:
use unboxed types everywhere. eg long instead of Long, int instead of Integer etc. Or eliminate the temporary variables